packages/radio/src/use-radio.ts

```diff
@@ -61,7 +63,7 @@ export interface UseRadioProps {
    */
   isInvalid?: boolean
   /**
-   * If `true`, the radio button will be invalid. This sets `aria-invalid` to `true`.
+   * If `true`, the radio button will be required. This sets `aria-invalid` to `true`.
```

Comment:
I think this still needs updating, compared to [<Input />'s isRequired prop](https://chakra-ui.com/docs/form/input#props)
"If `true`, the form control will required. This has 2 side effects: - The `FormLabel` will show a required indicator - The form element (e.g, Input) will have `aria-requred` set to `true`"

Reply:
Yeah you are right, want to make a quick PR for this?

Reply:
Done #2674